docs/asdf-astropy/example.rst

```diff
@@ -0,0 +1,119 @@
+.. _basic_example:
+
+=======
+Example
+=======
+
+
+In this example, we will show how to implement serialization for a new
+`~astropy.modeling.Model` object, but the basic principles apply to
+serialization of other ``astropy`` objects. As mentioned, adding a new object
+to ``asdf-astropy`` requires both a converter and a tagged-schema.
+
+All schemas for transforms are currently defined within the asdf-transform-schemas.
+Any new serializable transforms can have a corresponding new schema here. Let's
+consider a new model called ``MyModel``, a new model in ``astropy.modeling.functional_models``
+that has two parameters ``amplitude`` and ``x_0``. We would like to strictly require both
+of these parameters be set. We would also like to specify that these parameters can
+either be numeric type, or ``astropy.units.quantity`` type. A schema describing this
+model would look like
+
+.. code-block:: yaml
+
+    %YAML 1.1
+    ---
+    $schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+    id: "http://stsci.edu/schemas/asdf/transform/mymodel-1.0.0"
+    tag: "tag:stsci.edu:asdf/transform/mymodel-1.0.0"
```

Comment:
Let's remove this since we're no longer recommending use of the `tag:` validator at the top level